<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchWidget</string>
  </property>
  <widget class="QLabel" name="layerNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Layer Name</string>
   </property>
  </widget>
  <widget class="QListWidget" name="searchResultList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>571</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QLabel" name="attributeValueLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attribute Value</string>
   </property>
  </widget>
  <widget class="QComboBox" name="attriNameComboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="attributeValueInputLine">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="resultNumLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>共查询出 0 条结果</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
